modules/show.js

```diff
@@ -0,0 +1,42 @@
+const homeContainer = document.querySelector('.home-container');
+const listContainer = document.querySelector('.list-container');
+const addContainer = document.querySelector('.add-container');
+const contactContainer = document.querySelector('.contact-container');
+
+export default function show(name) {
```

Comment:
this one 🤓

Reply:
this is the only one I cant convert to arrow style.
because export default is not accepted with the arrow style, linter accepts export only.
I have tried to export at the end of the file but other functions do not recognize it as a function anymore "export default show();"